pennylane/gradients/__init__.py

```diff
@@ -111,7 +111,7 @@ def circuit(weights):
 
 .. code-block:: python
 
-    @qml.qnode(dev, gradient_fn=qml.gradients.param_shift)
+    @qml.qnode(dev, diff_method=qml.gradients.param_shift)

```

Comment:
Maybe I am wrong but I think it is falling back on backprop, the normal way of passing diff_method is the str: `parameter-shift` @lillian542 @timmysilv

Reply:
it does fall back on backprop because unless set, `diff_method` is `None` which is valid but probably not what the user intended. Our [qnode docs](https://docs.pennylane.ai/en/stable/code/api/pennylane.qnode.html) say you can set `diff_method` to be a string or an actual `gradient_transform`, and the qnode code handles the two equivalently. I've confirmed it works as expected.